tests/integration-tests/tests/schedulers/test_slurm/test_slurm/torque_job.sh

```diff
@@ -0,0 +1,2 @@
+#!/bin/sh
+echo "sleep 100"
```

Comment:
Any particular reason why `100` and not `60 ` for example?

Reply:
The script purpose is to test if we can submit the job by torque command. So we can use any word in `echo` script. Both `echo "sleep 100"` or `echo "sleep 60" ` will be executed in less than 1 second, so I just randomly selected a number.